### PR TITLE
broken_barh docstring

### PR DESCRIPTION
**Pull request by frankliuao**
## PR Summary
Fixing the incorrect information in the broken_barh docstring, which was discussed in issue #13768. 
**Error**
```
yranges : (*ymin*, *ymax*)
The y-position and extend for all the rectangles.
```
The text of the explanation is correct, however, the *ymax* should have been *yheight*, as it is the extend of the height of the bar. A user might be confused with *ymax*.

## PR Checklist

Following checklist are not applicable in this case. 
- [ ] Has Pytest style unit tests
- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: lib/matplotlib/axes/_axes.py [hunk 1]
```diff
@@ -2566,7 +2566,7 @@ def broken_barh(self, xranges, yrange, **kwargs):
             The x-positions and extends of the rectangles. For each tuple
             (*xmin*, *xwidth*) a rectangle is drawn from *xmin* to *xmin* +
             *xwidth*.
-        yranges : (*ymin*, *ymax*)
+        yranges : (*ymin*, *yheight*)
```
Comment (ImportanceOfBeingErnest) [inline lib/matplotlib/axes/_axes.py]: The argument is named `yrange` (without `s`).

Comment (frankliuao) [reply lib/matplotlib/axes/_axes.py]: Yes you are right! The original argument was wrong also. Let me do this again and also correct that. Do I need to open a new PR, or I can just use this if I push a new commit? @ImportanceOfBeingErnest

Comment (frankliuao) [reply lib/matplotlib/axes/_axes.py]: Done. Pushed commit. I think it's automatically updated. Power of GitHub!

**Comment by timhoffm** [comment]
Thanks, and congratulations on your first contribution to matplotlib!

**Pull request merged**